torch/nn/quantized/modules/conv.py

```diff
@@ -214,7 +225,8 @@ def from_float(cls, mod):
         act_scale, act_zp = activation_observer.calculate_qparams()
         assert weight_observer.dtype == torch.qint8, 'Weight observer must have a dtype of qint8'
         wt_scale, wt_zp = weight_observer.calculate_qparams()
-        bias_scale = float(wt_scale * act_scale)
+        bias_scale = float(act_scale / (2**16))
```

Comment:
Any reason to quantize bias using act_scale / 2**16 ? Wouldn't quantizing it with just act_scale work?

Reply:
This was done to use as much precision for bias as possible. If we use act_scale, bias would be quantized to 8 bits, instead of 32 bits.

Reply:
act_scale (aka output_scale) quantizes output_act to 8-bits. We assume that bias is on the same range as output so act_scale will quantize bias to 8-bit as well? Is this the reasoning? Just asking for my understanding.

Reply:
Yes, thats my reasoning. bias should be comparable to the raw conv outputs and therefore the output scale is a proxy for the bias scale itself